pkg/httpproxy/proxy.go

```diff
@@ -104,6 +104,10 @@ func (p *proxy) proxy(req *http.Request) error {
 			continue
 		}
 
+		if strings.HasPrefix(name, "Impersonate-") {
```

Comment:
lets add the concrete headers we are dropping, to the https://github.com/rancher/rancher/blob/ba1ee92cb6260b261a75b40a047ad51ac98e416a/pkg/httpproxy/proxy.go#L22